### PR TITLE
Better status icon alignment in the conversation list

### PR DESCRIPTION
**Pull request by makingthematrix**
#### APK
[Download build #9987](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9987/artifact/build/artifact/wire-dev-PR1338-9987.apk)
[Download build #10038](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10038/artifact/build/artifact/wire-dev-PR1338-10038.apk)
[Download build #10043](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10043/artifact/build/artifact/wire-dev-PR1338-10043.apk)
[Download build #10046](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10046/artifact/build/artifact/wire-dev-PR1338-10046.apk)
[Download build #10048](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10048/artifact/build/artifact/wire-dev-PR1338-10048.apk)
[Download build #10049](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10049/artifact/build/artifact/wire-dev-PR1338-10049.apk)
[Download build #10050](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10050/artifact/build/artifact/wire-dev-PR1338-10050.apk)
[Download build #10053](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10053/artifact/build/artifact/wire-dev-PR1338-10053.apk)

### CODE REVIEW

**Comment by someoneEsle** [comment]
Bug 1
It is not aligned correctly (still too high). Same for the icon in the contacts list, and in all the other lists we display (e.g. when you open the list of all the people who liked a message, or when you add participants to a group chat)

**Comment by someoneEsle** [comment]
Eike created a ticket for this with more complete information :)

File: app/src/main/scala/com/waz/zclient/common/views/ChatheadWithTextFooter.scala [hunk 1]
```diff
@@ -40,115 +39,83 @@ class ChatheadWithTextFooter(val context: Context, val attrs: AttributeSet, val
...
-    updateView(data)
-  }
-  if (ThemeUtils.isDarkTheme(context)) {
-    applyDarkTheme()
-  } else {
-    applyLightTheme()
-  }
+  userInfo.on(Threading.Ui) { updateView }
+
+  if (ThemeUtils.isDarkTheme(context)) applyDarkTheme() else applyLightTheme()
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/common/views/ChatheadWithTextFooter.scala]: in the future, keep in mind that `ThemeUtils` is a legacy class - use `ThemeController.scala` instead

File: app/src/main/scala/com/waz/zclient/common/views/ChatheadWithTextFooter.scala [hunk 2]
```diff
@@ -40,115 +39,83 @@ class ChatheadWithTextFooter(val context: Context, val attrs: AttributeSet, val
...
 
+  def applyLightTheme(): Unit = footer.setTextColor(ContextCompat.getColor(getContext, R.color.text__primary_light))
+  def applyDarkTheme(): Unit = footer.setTextColor(ContextCompat.getColor(getContext, R.color.text__primary_dark))
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/common/views/ChatheadWithTextFooter.scala]: these last three lines could be replaced with:

```
import ContextUtils._
...

footer.setTextColor(getColor(if (themes.isDark) R.color.text__primary_light else R.color.text__primary__dark)).
```

Comment (deanrobertcook) [reply app/src/main/scala/com/waz/zclient/common/views/ChatheadWithTextFooter.scala]: (notice, also use `ContextUtils` rather than `ContextCompat`)

**Comment by makingthematrix** [comment]
Unused imports removed

**Pull request merged**